Essentials/src/main/resources/messages.properties

```
@@ -232,9 +232,23 @@ destinationNotSet=Destination not set\!
 disabled=disabled
 disabledToSpawnMob=\u00a74Spawning this mob was disabled in the config file.
 disableUnlimited=\u00a76Disabled unlimited placing of\u00a7c {0} \u00a76for\u00a7c {1}\u00a76.
+discordCommandAccountArgumentUser=The discord account to check the minecraft of
+discordCommandAccountDescription=Checks the minecraft account of either you or another user if provided
+discordCommandAccountResponseLinked=Your account is linked to the minecraft account: **{0}**
+discordCommandAccountResponseLinkedOther={0}'s account is linked to the minecraft account: **{1}**
+discordCommandAccountResponseNotLinked=You do not have a minecraft account linked.
+discordCommandAccountResponseNotLinkedOther={0} does not have a minecraft account linked.
 discordCommandExecuteDescription=Executes a console command on the Minecraft server.
 discordCommandExecuteArgumentCommand=The command to be executed
 discordCommandExecuteReply=Executing command: "/{0}"
+discordCommandUnlinkDescription=Unlinks any associated minecraft account from this discord account
+discordCommandUnlinkInvalidCode=You do not currently have a minecraft account linked to this discord account!
+discordCommandUnlinkUnlinked=Unlinked this discord account from all associated minecraft accounts.
+discordCommandLinkArgumentCode=The code associated with your minecraft account as provided in game
+discordCommandLinkDescription=Links your discord account with the minecraft account associated with the given code
+discordCommandLinkHasAccount=You already have an account linked, to unlink your current account do /unlink
+discordCommandLinkInvalidCode=Invalid Code!
+discordCommandLinkLinked=Successfully linked your account!
```

```suggestion
discordCommandUnlinkDescription=Unlinks the Minecraft account currently linked to your Discord account
discordCommandUnlinkInvalidCode=You do not currently have a Minecraft account linked to Discord!
discordCommandUnlinkUnlinked=Your Discord account has been unlinked from all associated Minecraft accounts.
discordCommandLinkArgumentCode=The code provided in-game to link your Minecraft account
discordCommandLinkDescription=Links your Discord account with your Minecraft account using a code from the in-game /TODO-AAAAAAAAA command
discordCommandLinkHasAccount=You already have an account linked! To unlink your current account, do /unlink.
discordCommandLinkInvalidCode=Invalid linking code! Make sure you've run /TODO-AAAAAAAA in-game and copied the code correctly.
discordCommandLinkLinked=Successfully linked your account!
```

* Maybe `discordCommandUnlinkUnlinked` -> `discordCommandUnlinkSuccess` for clarity?
* TODO should this refer to one Discord to multiple MC accounts? I've suggested these as if one-to-one but rewrite if it's one-to-many